src/components/App/App.test.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+
+import App from './App';
+
+describe('App', () => {
+  let instance;
+  beforeEach(() => {
+    instance = render(<App />);
+  });
+
+  it('renders the title', () => {
+    expect(instance.getByText('Shopping List')).toBeInTheDocument();
+  });
+
+  it('renders the number of items on the list', () => {
+    expect(instance.getByText(/[0-9]+ items/)).toBeInTheDocument();
```

Comment:
This is an example of the regex selector that I mentioned a few days ago, it's useful here because we don't know / care how many items there are, we just want to check there's a number, you can see an example of this here:
https://regex101.com/r/G9LDg9/1